docs/guides/expectations/creating_custom_expectations/how_to_create_custom_table_expectations.md

```diff
@@ -285,15 +285,14 @@ Completeness checklist for ExpectTableColumnsToBeUnique:
 
 ### 8. Linting
 
-Finally, we need to lint our now-functioning Custom Expectation. Our CI system will test your code using `black`, `isort`, `flake8`, and `pyupgrade`. 
+Finally, we need to lint our now-functioning Custom Expectation. Our CI system will test your code using `black`, `isort`, and `ruff`.
```

Comment:
Looks like we use this snippet a lot - not import here but let's maybe find a way to reuse this? cc @Rachel-Reverie I think we have the ability to save snippets through our own MDX right?